### PR TITLE
Removing permission of 'Administration acl' from access control field in add/edit user screen

### PR DESCRIPTION
**Pull request by matrix-amiel**
Hi.
This PR is fix for bug https://github.com/openemr/openemr/issues/1352

Thanks
Amiel

### CODE REVIEW

File: interface/usergroup/user_admin.php [hunk 1]
```diff
@@ -420,7 +420,7 @@ function authorized_clicked() {
 
 <?php
  // Collect the access control group of user
-if (isset($phpgacl_location) && acl_check('admin', 'acl')) {
+if (isset($phpgacl_location)) {
```
Comment (bradymiller) [inline interface/usergroup/user_admin.php]: this entire if should be removed

File: interface/usergroup/usergroup_admin.php [hunk 2]
```diff
@@ -196,7 +196,7 @@
             sqlStatement("update users set weno_prov_id = '$erxprid' where id = ? ", array($_POST["id"]));
         }
       
-        if (isset($phpgacl_location) && acl_check('admin', 'acl')) {
+        if (isset($phpgacl_location)) {
```
Comment (bradymiller) [inline interface/usergroup/usergroup_admin.php]: this entire if should be removed

File: interface/usergroup/usergroup_admin_add.php [hunk 3]
```diff
@@ -334,7 +334,7 @@ function authorized_clicked() {
 
 <?php
  // List the access control groups if phpgacl installed
-if (isset($phpgacl_location) && acl_check('admin', 'acl')) {
+if (isset($phpgacl_location)) {
```
Comment (bradymiller) [inline interface/usergroup/usergroup_admin_add.php]: this entire if should be removed

File: interface/usergroup/usergroup_admin.php [hunk 4]
```diff
@@ -290,7 +290,7 @@
                   sqlStatement("insert into groups set name = '" . trim(formData('groupname')) .
                     "', user = '" . trim(formData('rumple')) . "'");
 
-                if (isset($phpgacl_location) && acl_check('admin', 'acl') && trim(formData('rumple'))) {
+                if (isset($phpgacl_location) && trim(formData('rumple'))) {
```
Comment (bradymiller) [inline interface/usergroup/usergroup_admin.php]: remove the isset($phpgacl_location) from this if

File: interface/usergroup/user_admin.php (hunk 1 above)
Comment (matrix-amiel) [reply interface/usergroup/user_admin.php]: Done

File: interface/usergroup/usergroup_admin.php (hunk 2 above)
Comment (matrix-amiel) [reply interface/usergroup/usergroup_admin.php]: Done.

File: interface/usergroup/usergroup_admin_add.php (hunk 3 above)
Comment (matrix-amiel) [reply interface/usergroup/usergroup_admin_add.php]: Done.

File: interface/usergroup/usergroup_admin.php (hunk 4 above)
Comment (matrix-amiel) [reply interface/usergroup/usergroup_admin.php]: Done.

**Comment by bradymiller** [comment]
hi @matrix-amiel ,
Code looks good. If this is testing well for you, then let me know, and then I'll bring it into the codebase.
-brady

**Comment by matrix-amiel** [comment]
Yes.
I checked it. working well.

Thanks.

**Comment by bradymiller** [comment]
Great! I'll fix the merge conflict this evening and then bring it into the codebase.

**Comment by bradymiller** [comment]
Just brought this in. Thanks for the fix! -brady

**Pull request merged**